recipes/fastlinkingnumbers/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "fastlinkingnumbers" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/antequ/fastlinkingnumbers/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b3393bffb8669ab8214b1b86ae8b1d48abd70ba021134a3f2a5e5fcfd641e7d9
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - cmake
+    - eigen
+    - libgomp
```

Comment:
Why are these `build` dependencies? Usually such libraries are in the `host` section.

Reply:
I've moved `eigen` to `host`, but it seems like this `libgomp` is needed in addition to the `openmp` in `host` for things to work, and the [knowledge base](https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp) seems to say to put it in `build`?

Reply:
Ok. I was not aware of that but this makes sense then for libgomp.